src/open_inwoner/scss/views/App.scss

```diff
@@ -3,6 +3,7 @@
 
   --container-width: 1024px;
   --mobile-s-width: 768px;
+  --desktop-hd-width: 1600px;
```

Comment:
This can be removed right? Or is it there in order to have an order with all kinds of resolutions?